ext/vscode/src/utils/process.ts

```diff
@@ -41,6 +67,10 @@ export async function spawnAsync(
                 cancellationListener = undefined;
             }
 
+            if (authServer) {
+                authServer.close();
+            }
```

Comment:
Equally valid but a little more brief, applies to some other places as well:

```suggestion
            authServer?.close();
```

Reply:
Did a pass and adopted this style.